### PR TITLE
Merge `new project` templates into one

### PR DESCRIPTION
**Pull request by delvh**
Additionally simplify the `new project` template slightly.

Review hint: Disable whitespace changes.

<details><summary>Before</summary>

## New repo project
grafik

## Edit repo project
grafik

## New user/org project
grafik

## Edit user/org project
grafik
</details>

<details><summary>After</summary>

## New repo project
grafik

## Edit repo project
grafik

## New user/org project
grafik

## Edit user/org project
grafik
</details>

### CODE REVIEW

File: templates/org/projects/new.tmpl
Comment (silverwind) [inline templates/org/projects/new.tmpl]: `milestone`?

Comment (silverwind) [reply templates/org/projects/new.tmpl]: I guess this is from bad copy-paste CSS that came with initial projects PR, but still something we should eventually clean up, so not blocking on it.

File: routers/web/org/projects.go
Comment (lunny) [inline routers/web/org/projects.go]: Maybe `RenderNewProject` is a better name?

Comment (delvh) [reply routers/web/org/projects.go]: I am fine with both.
@silverwind, I'll let you decide as the third voice.

Comment (silverwind) [reply routers/web/org/projects.go]: I do think I prefer `Render` prefix, `NewProject` alone sounds unclear.

**Comment by delvh** [comment]
@lunny @silverwind all done.

**Review by silverwind — APPROVED** [review]

**Comment by delvh** [comment]
:astonished: I just noticed I can remove a lot of dead code.
Commit incoming!

**Comment by silverwind** [comment]
> 😲 I just noticed I can remove a lot of dead code. Commit incoming!

image

**Comment by delvh** [comment]
According to my quick calculation, this PR reduces Giteas binary and RAM size by roughly `11.4kb (deletions) - 4.6kb (additions) = 6.8kb (savings)` :tada:

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**
